### PR TITLE
Add shellcheck tools and modify copyright hook

### PR DESCRIPTION
**Pull request by gongweibao**
### PR types
New features

### PR changes
 Others

### Describe
1. 以前的copyright在.sh文件中是不加的，现在加上了。Add shellcheck tools 
2. sh之前没有check，现在用shellcheck做静态检查。Modify copyright hook

### How to fix large files:
https://github.com/PaddlePaddle/Paddle/pull/27905

### CODE REVIEW

File: tools/codestyle/copyright.hook [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
-import platform
+import io
+import re
+import sys
+import os
+import datetime
 
-COPYRIGHT = '''
-Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
+COPYRIGHT = '''Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```
Comment (lanxianghit) [inline tools/codestyle/copyright.hook:26]: 2016？

Comment (gongweibao) [reply tools/codestyle/copyright.hook:26]: Will be replaced by the current year in running.

File: tools/manylinux1/Dockerfile.cuda10_cudnn7_gcc8_ubuntu16 [hunk 2]
```diff
@@ -26,7 +26,7 @@ RUN rm -rf /temp_gcc82 && rm -rf /gcc-8.2.0.tar.xz && rm -rf /gcc-8.2.0
 RUN apt-get update && \
     apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
     libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
-    xz-utils tk-dev libffi-dev liblzma-dev openmpi-bin openmpi-doc libopenmpi-dev
+    xz-utils tk-dev libffi-dev liblzma-dev openmpi-bin openmpi-doc libopenmpi-dev shellcheck
```
Comment (luotao1) [inline tools/manylinux1/Dockerfile.cuda10_cudnn7_gcc8_ubuntu16:29]: 除了CI上的镜像要添加shellcheck，开发镜像也要添加 @tianshuo78520a 请帮忙看下开发镜像要添加到哪里

Comment (gongweibao) [reply tools/manylinux1/Dockerfile.cuda10_cudnn7_gcc8_ubuntu16:29]: Done.

**Review by lanxianghit — APPROVED** [review]

**Review by luotao1 — APPROVED** [review]
LGTM

**Review by kolinwei — APPROVED** [review]

**Review by chalsliu — APPROVED** [review]

**Pull request merged**